frontend/views/containers/chatroom/MessageInteractive.vue

```diff
@@ -37,10 +37,38 @@ const interactiveMessage = (proposal, baseOptions = {}) => {
   const options = Object.assign({}, baseOptions)
   const { proposalType, proposalData, variant } = proposal
 
+  const groupSettingType = proposalData.setting
+  let proposalSettingType
+  if (proposalData.ruleName !== proposalData.current.ruleName) {
+    proposalSettingType = 'votingSystem'
+  } else if (proposalData.ruleThreshold !== proposalData.current.ruleThreshold) {
+    proposalSettingType = 'votingRule'
+  }
+
   if (proposalType === PROPOSAL_GENERIC) {
     options['title'] = proposalData.name
+  } else if (proposalType === PROPOSAL_GROUP_SETTING_CHANGE) {
+    if (groupSettingType === 'mincomeAmount') {
+      options['setting'] = L('mincome')
+    } else if (groupSettingType === 'distributionDate') {
+      options['setting'] = L('distribution date')
+    }
+  } else if (proposalType === PROPOSAL_PROPOSAL_SETTING_CHANGE) {
+    if (groupSettingType === 'votingSystem') {
+      options['setting'] = L('voting system')
+    } else if (groupSettingType === 'votingRule') {
+      options['setting'] = L('voting rules')
+    }
   }
 
+  const settingChangeMessages = (options) => ({
+    [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to change the groups {setting}.', options),
+    [PROPOSAL_VARIANTS.EXPIRING]: L('Proposal from {from} to change the {setting} is expiring.', options),
+    [PROPOSAL_VARIANTS.ACCEPTED]: L('The groups {setting} changed.'),
+    [PROPOSAL_VARIANTS.REJECTED]: L('The group {setting} hasn\'t changed.'),
+    [PROPOSAL_VARIANTS.EXPIRED]: L('The group {setting} hasn\'t changed.')
```

Comment:
It's generally preferred to use double-quotes whenever the string has `'` inside of it